programs/grpc-client/grpc_client.cpp

```diff
@@ -0,0 +1,94 @@
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include <grpc++/channel.h>
+#include <grpc++/client_context.h>
+#include <grpc++/create_channel.h>
+#include <grpc++/security/credentials.h>
+#include "GrpcConnection.grpc.pb.h"
+
+class GRPCClient {
```

Comment:
As I already said, please convert this file into an integration unit-test.
It seems it must not be difficult, see https://grpc.io/docs/quickstart/python/